backend/probloom/prob/views.py

```diff
@@ -1184,8 +1193,7 @@ def solve_problem(request: HttpRequest, p_id: int) -> HttpResponse:
     .. code-block:: typescript
 
        interface SolveProblemResponseSuccess {
-         result: 'SUCCESS';
-         correct: boolean;
+         result: boolean;
```

Comment:
Please note that interface for `SolveProblemResponseSuccess` has been changed, since it was a bit confusing.